forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -354,31 +354,49 @@ def get(application_id: int):
 
 
 @cors_preflight("POST,OPTIONS")
-@API.route("/form-create", methods=["POST", "OPTIONS"])
+@API.route("/formio", methods=["POST", "OPTIONS"])
```

Comment:
Hmm.. may be make it generic? Say `/design` or something else than `formio` ? Since we are providing an abstraction, it's to better to keep it generic so that in case if we replace formio with something else in future, we dont need to rename the endpoints again ?

Reply:
thank you for your comments